test/unit-tests/handlers/targetHandler.test.js

```diff
@@ -14,6 +16,17 @@ describe('TargetHandler', () => {
       targetHandler.__set__('cri', mockCri);
     });
 
+    after(() => {
+      const createdSessionListener = targetHandler.__get__('createdSessionListener');
+      targetHandler
+        .__get__('eventHandler')
+        .removeListener('createdSession', createdSessionListener);
+      targetHandler = rewire('../../../lib/handlers/targetHandler');
+      targetHandler
```

Comment:
why do this in `after`? I see this in other files too. I mean, removing listener on the mock?

Reply:
unlike require each time rewire is called it creates a new instance and attaches the listener too... hence gotta remove them so that it does not affect other tests
https://github.com/jhnns/rewire#caveats